catalog-info.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "payment"
```

Does this name need to be more descriptive? May be a bit too fussy lol

```suggestion
  name: "ccpay-payment"
```

---

```suggestion
  name: "payment-api"
```